link/src/main/java/com/stripe/android/link/ui/menus/LinkMenu.kt

```diff
@@ -0,0 +1,80 @@
+package com.stripe.android.link.ui.menus
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.stripe.android.link.theme.HorizontalPadding
+import com.stripe.android.link.theme.MinimumTouchTargetSize
+import com.stripe.android.link.theme.linkColors
+
+/**
+ * An item to be displayed in a [LinkMenu].
+ *
+ * @property textResId The resource ID of the text of the item
+ * @property destructive Whether this item should be rendered with the error text color
+ */
+internal interface LinkMenuItem {
+    val textResId: Int
```

Comment:
Can annotate with `@StringRes`

Reply:
I wanted to do the same, but that doesn’t work for interface properties.